tests/aws/services/cloudformation/resources/test_lambda.snapshot.json

```diff
@@ -1606,5 +1606,60 @@
         "LayerVersionRef": "arn:<partition>:lambda:<region>:111111111111:layer:<layer-name:1>:1"
       }
     }
+  },
+  "tests/aws/services/cloudformation/resources/test_lambda.py::test_lambda_logging_config": {
+    "recorded-date": "03-04-2025, 17:34:50",
+    "recorded-content": {
+      "stack_resource_descriptions": {
+        "StackResources": [
+          {
+            "DriftInformation": {
+              "StackResourceDriftStatus": "NOT_CHECKED"
+            },
+            "LogicalResourceId": "LambdaFunction",
+            "PhysicalResourceId": "<function-name>",
+            "ResourceStatus": "CREATE_COMPLETE",
+            "ResourceType": "AWS::Lambda::Function",
+            "StackId": "arn:<partition>:cloudformation:<region>:111111111111:stack/<stack-name:1>/<resource:1>",
+            "StackName": "<stack-name:1>",
+            "Timestamp": "timestamp"
+          },
+          {
+            "DriftInformation": {
+              "StackResourceDriftStatus": "NOT_CHECKED"
+            },
+            "LogicalResourceId": "MyFnServiceRole",
+            "PhysicalResourceId": "<stack-name:1>-MyFnServiceRole-IVwvF4i3Fcqy",
```

Comment:
`IVwvF4i3Fcqy` looks like a random uuid. Does this test pass when validated against AWS without updating the snapshot? A transformer for the Lambda role could help here and potentially resolve the TODO in `skip_snapshot_verify`.

Reply:
I see that's a common issue in Lambda CloudFormation tests; not nice but feel free to skip :)

Reply:
I didn't like it either, and now I understand a bit better why that happened. Now it's fixed by setting `reference_replacement=False` to that property. Thanks for pointing it out!